apps/web/screens/Article.tsx

```diff
@@ -340,11 +349,20 @@ export interface ArticleProps {
 }
 
 const ArticleScreen: Screen<ArticleProps> = ({ article, namespace }) => {
+  const { setContentfulId } = useContext(GlobalContext)
   const n = useNamespace(namespace)
   const { query } = useRouter()
   const { activeLocale } = useI18n()
   const { makePath } = routeNames(activeLocale)
 
+  useEffect(() => {
```

Comment:
Could we make a custom hook for this? Like useContentfulId rather than using useEffect everywhere and having to remember to cleanup

Reply:
True, I moved it into a hook.